### PR TITLE
[WIP] Willbasky/fix command

### PR DESCRIPTION
**Pull request by willbasky**

### CODE REVIEW

File: src/LambdaLauncher/Plugins/Sway.hs [hunk 1]
```diff
@@ -31,17 +31,17 @@ instance ToJSON Node
...
 windowToResults :: Window -> LambdaLauncher.Types.Result
 windowToResults (Window name app_id id pid) =
   Action name 2 $ callProcess "swaymsg" ["[con_id=" ++ show id ++ "] focus"]
 
 sway :: Plugin
 sway s = do
-  tree <- encodeUtf8 <$> pack <$> readProcess "swaymsg" ["-t", "get_tree"] ""
-  return $
-    concat $
-    map windowToResults <$> filter (\(Window name _ _ _) -> isInfixOf s name) <$>
```
Comment (willbasky) [inline src/LambdaLauncher/Plugins/Sway.hs:45]: It is better to make labels for Window type and use here `Window{..}`

Comment (willbasky) [reply src/LambdaLauncher/Plugins/Sway.hs:45]: I don`t know what fields mean to make it myself.

Comment (balsoft) [reply src/LambdaLauncher/Plugins/Sway.hs:45]: Good point. It probably should be 
```
data Window =
  Window 
         { name :: String
         , app_id :: Maybe String
         , id :: Integer
         , pid :: Integer }
```

Comment (willbasky) [reply src/LambdaLauncher/Plugins/Sway.hs:45]: Ok, i will add.

**Comment by balsoft** [comment]
Looks good. Add the `Window` labels and it's merged!

**Comment by balsoft** [comment]
All good.

**Pull request merged**